documentation/content/en/books/handbook/zfs/_index.adoc

```diff
@@ -1331,12 +1331,14 @@ For legacy boot using GPT, use the following command:
 # gpart bootcode -b /boot/pmbr -p /boot/gptzfsboot -i 1 ada1
```

Comment:
`ada1` was assumed in 043d1f2004aee201ba00f65cffaab7e6feb3c31b (2018), maybe better now to assume `ada0` here and below. 

(A separate pull request, to keep this one simple. [261212 – Update the ZFS chapter of the FreeBSD Handbook, and other OpenZFS-related pages](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=261212) is _very_ broad.)

Reply:
as you wish.